src/kepler/tests.zig

```diff
@@ -0,0 +1,306 @@
+const std = @import("std");
+const os = @import("root").os;
+const kepler = os.kepler;
+
+fn rdtsc() u64 {
+    var eax: u32 = undefined;
+    var edx: u32 = undefined;
+    asm volatile ("rdtsc"
+        : [_] "={eax}" (eax),
+          [_] "={edx}" (edx)
+    );
+    return @as(u64, eax) + (@as(u64, edx) << 32);
+}
```

Comment:
Move this into `x86_64.zig`, can be useful for other code :+1:

Reply:
hm, I forgot that I had platform-dependent code hardcoded in test. I guess I should remove it somehow

Reply:
If it makes things easier, I can add the aarch64 equivalent so that you can call a platform abstracted version